pkg/utils/asynciter.go

```diff
@@ -139,12 +144,12 @@ func (ic *AsyncItemsCursor) NextItem() (v3io.Item, error) {
 	// Read response from channel
 	resp := <-ic.responseChan
 	if resp.Error != nil {
+		fmt.Println("error reading from response channel:", resp, "error", resp.Error, "request:", resp.Request().Input)
```

Comment:
Use logger?

Reply:
`ic.logger.Info(...)` (Or some other log level), although it would probably better to log this higher up the stack.